src/test/java/teammates/test/cases/action/AdminActivityLogPageActionTest.java

```diff
@@ -381,53 +379,52 @@ public void removeAndRestoreManyLogs() {
         gaeSimulation.loginAsAdmin("admin");
         gaeSimulation.clearLogs();
 
-        Date today = TimeHelper.getDateOffsetToCurrentTime(0);
+        Instant today = TimeHelper.getInstantDaysOffsetFromNow(0);
```

Comment:
I didn't use the member variable here because this is necessary for the logic of the test. Basically the test expect this `today` to be before the `today` in the next function. This `today` is the reference point for the logs. The `today` in the next function is the search reference point. Because searching is not inclusive, so having this `today` before search reference `today` kind of makes the search to be as if it's inclusive.

Reply:
Can just delete this variable altogether and use `Instant.now().toEpochMilli()` below?